### ISSUE TITLE
themet museum broken

### ISSUE THREAD

**Issue by CookieSalesman** [uBlockOrigin/uAssets#13792]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`https://store.metmuseum.org/stationery-art-supplies/notecards-correspondence`

### Describe the issue

Using Chrome, which I never touch. Ublock should not have any modifications to it.

Not even signed in on chrome. I have raspberry pi running a very very conservative blocklist.

Basically, this page `https://store.metmuseum.org/stationery-art-supplies/notecards-correspondence`
doesn't load any of the pictures at all. Two loading loop icons just spin and nothing loads.

### Screenshot(s)

![image](https://user-images.githubusercontent.com/9441933/174689246-c95d3520-8d21-47b9-a004-ea0cab9f63c3.png)

### uBlock Origin version

uBlock Origin 1.43.0

### Browser name and version

Chrome latest 2022-06-20

### Settings

- No significant changes to report.

**Comment by CookieSalesman** [comment]
I put that in My Filters. Adding that and refreshing the page made the pictures load.

I mean I see that this worked although I'm not sure if you meant that this was the best fix for everyone who would come across the issue?

**Comment by stephenhawk8054** [comment]
> I put that in My Filters. Adding that and refreshing the page made the pictures load.
> 
> I mean I see that this worked although I'm not sure if you meant that this was the best fix for everyone who would come across the issue?

If it works, I'll add it to the main list.
